### PR TITLE
feat : cart management

### PR DESCRIPTION
**Pull request by aqsaaqeel**
features : 
 Add to Cart

[thrifters-cart.netlify.app](https://thrifters-cart.netlify.app/)

You can view the Products page by clicking on any card on landing page and it will get added to cart.

Will add remove from cart, increase and decrease quantity in next PR

### CODE REVIEW

File: src/Components/Card/Card.jsx [hunk 1]
```diff
@@ -1,75 +1,93 @@
...
-                </div>
+          <div className="card-footer horizontal">
+            <div className="price">
+              <div className="card-subheading">New Arrivals</div>
+              <div className="small-space"></div>
             </div>
+            <div className="small-space"></div>
+            <div className="card-heading">{props.collection}</div>
+            <div className="card-subheading detail">
+              Get your favorite summer clothes at affordable prices
```
Comment (Vivek-Py) [inline src/Components/Card/Card.jsx:32]: Just a suggestion maintain a single file for stray strings. It becomes easy to maintain.  For example, if you want to change text in a component you won't have to go through the whole component instead you can go through constants file where you've stored all stary strings. Not necessary but good to have.

**Review by Vivek-Py — APPROVED** [review]

**Review by devananth — APPROVED** [review]
Code looks good !

File: src/Components/Card/Card.jsx (hunk 1 above)
Comment (aqsaaqeel) [reply src/Components/Card/Card.jsx:32]: Thank you for the review, will do that in the upcoming PRs

Comment (Vivek-Py) [reply src/Components/Card/Card.jsx:32]: No worries, happy to help!

**Pull request merged**